local-cli/upgrade/upgrade.js

```diff
@@ -57,6 +57,17 @@ module.exports = function upgrade(args, config) {
             );
             return Promise.resolve();
           }
+          
+          if (semver.satisfies(v, '~0.26.0')) {
```

Comment:
No, but I am assuming if one will be upgrading to 0.27 we will provide a new version of rnpm-plugin-upgrade to make it consistent and easier to work with (or maybe change it completely? I am not settled on the interface yet to be honest)